hack/setup_env.sh

```diff
@@ -53,11 +53,11 @@ function assisted_service() {
     install -o root -g root -m 0755 /tmp/kubectl /usr/local/bin/kubectl && \
     rm -f /tmp/kubectl
 
+  yum clean all && yum makecache
+  yum --disablerepo=epel -y update ca-certificates
```

Comment:
what ``--disablerepo=epel`` means in the context of this command?

Reply:
iinm, it's just a common workaround to update certs, as epel repo is using https. I.e. update the certificates while avoiding failure with fetching epel.

Reply:
so we update certs for all repositories but epel, and afterwards it will not try to do the same for epel?
any link to this kind of suggestion? (from stackoverflow I assume)

Reply:
I think it should update all certs, the disable of epel is just to prevent failure.
here's some reference: https://techglimpse.com/yum-install-epel-repo-error-solution/

Reply:
can we see if we can just remove epel usage?
that would be better if possible

Reply:
epel is required for p7zip: https://linuxconfig.org/how-to-install-p7zip-on-redhat-8
do u suggest install it manually instead? is it worth the effort:/

Reply:
I couldn't find any 7za usage
So our best option is to remove installation of epel and 7zip

Reply:
no luck. jq is also in epel, not sure if it's used in the image though.

Reply:
you're sure ``jq`` requires epel?
I did manage to remove epel installation and still install ``jq``

Reply:
tried 'skipper build assisted-service-build' without jq:
#5 9.056 No package jq available.

maybe you got jq from another repo?
try 'yum search jq'?

Reply:
jq is needed for the scripts

Reply:
Ah oops, you're right. It's required
Either way we can proceed with your solution if it will do no harm (and it seems to not harm anything)

Reply:
we can move the certificate update to the place in the Dockerfile where we bring in epel release. This way it will also affect previous calls to yum update

Reply:
The reason it's so confusing is that we're based on centos **7** in our dockerfile
We should figure out a way to solve it too, but that's another issue